docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
 
-**pids_limit**=""
-  Maximum number of processes allowed in a container (default: 1024)
+**selinux**=false
+  If true, SELinux will be used for pod separation on the host.
 
-**runtime**=""
-  OCI runtime path (default: "/usr/bin/runc")
+**seccomp_profile**="/etc/crio/seccomp.json"
+  Path to the seccomp.json profile which is used as the default seccomp profile for the runtime.
 
-**selinux**=*true*|*false*
-  Enable selinux support (default: false)
+**apparmor_profile**=""
+  Used to change the name of the default AppArmor profile of CRI-O. The default profile name is "crio-default-" followed by the version string of CRI-O.
+
+**cgroup_manager**="cgroupfs"
+  Cgroup management implementation used for the runtime.
+
+**default_capabilities**=[]
+  List of default capabilities for containers. If unset (e.g., commented out), the default list of capabilities defined in the spec will be added. If `capabilities` is empty (i.e., set to `[]`), only the capabilities defined in the container json file by the user/kube will be added.
+
+  The default list looks as follows:
+```
+  default_capabilities = [
+          "CHOWN",
+          "DAC_OVERRIDE",
+          "FSETID",
+          "FOWNER",
+          "NET_RAW",
+          "SETGID",
+          "SETUID",
+          "SETPCAP",
+          "NET_BIND_SERVICE",
+          "SYS_CHROOT",
+          "KILL",
+  ]
+```
+
+**hooks_dir_path**="/usr/share/containers/oci/hooks.d"
+  Path to the OCI hooks directory for automatically executed hooks.
 
-**signature_policy**=""
-  Path to the signature policy json file (default: "", to use the system-wide default)
+**default_mounts**=[]
+  List to default mounts for each container. **Deprecated:** this option will be removed in future versions in favor of `default_mounts_file`.
 
-**seccomp_profile**=""
-  Path to the seccomp json profile to be used as the runtime's default (default: "/etc/crio/seccomp.json")
+**default_mounts_file**="/etc/containers/mounts.conf"
+  Path to the file specifying the defaults mounts for each container. The format of the config is /SRC:/DST, one mount per line. Notice that CRI-O reads its default mounts from the following two files:
 
-**apparmor_profile**=""
-  Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
+    1) `/etc/containers/mounts.conf` (i.e., default_mounts_file): This is the override file, where users can either add in their own default mounts, or override the default mounts shipped with the package.
 
-**no_pivot**=*true*|*false*
-  Instructs the runtime to not use pivot_root, but instead use MS_MOVE
+    2) `/usr/share/containers/mounts.conf`: This is the default file read for mounts. If you want CRI-O to read from a different, specific mounts file, you can change the default_mounts_file path right below. Note, if this is done, CRI-O will only add mounts it finds in this file.
```

Comment:
The default_mounts_file path is not "right below" in this doc.  I'd remove "right below".

Reply:
:+1: